src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private int localyear;
```

Comment:
Якщо в назві кілька слів, прийнято писати їх в camelCase.
Також, в даному випадку буде достатньо просто `year`, а приставка `local` не несе в собі жодного сенсового навантаження
```suggestion
    private int year;
```

Reply:
updated to be year
thank you for explanation!